node/pkg/governor/governor_monitoring.go

```diff
@@ -244,62 +245,107 @@ func (gov *ChainGovernor) resetReleaseTimerForTime(vaaId string, now time.Time,
-			return 0, err
+			return 0, 0, 0, err
+		}
+		if t.value < 0 {
+			// If a transfer is negative then it is an incoming, flow-cancelling transfer.
+			// We can use the dbTransfer.Value for calculating the sum because it is the unsigned version
+			// of t.Value
+			flowCancelNotional += t.dbTransfer.Value
+		} else {
+			smallTxOutgoingNotional += t.dbTransfer.Value
 		}
-		sum = checkedSum
-	}
-
-	// Do not return negative values. Instead, saturate to zero.
-	if sum <= 0 {
-		return 0, nil
 	}
 
-	return uint64(sum), nil
+	return netNotional, smallTxOutgoingNotional, flowCancelNotional, nil
 }
 
-// REST query to get the current available notional value per chain.
+// REST query to get the current available notional value per chain. This is defined as the sum of all transfers
+// subtracted from the chains's dailyLimit.
+// The available notional limit by chain represents the remaining capacity of a chain. As a result, it should not be
+// a negative number: we don't want to represent that there is "negative value" available.
 func (gov *ChainGovernor) GetAvailableNotionalByChain() (resp []*publicrpcv1.GovernorGetAvailableNotionalByChainResponse_Entry) {
 	gov.mutex.Lock()
 	defer gov.mutex.Unlock()
 
 	startTime := time.Now().Add(-time.Minute * time.Duration(gov.dayLengthInMinutes))
-	for _, ce := range gov.chains {
-		value, err := sumValue(ce.transfers, startTime)
+
+	// Iterate deterministically by accessing keys from this slice instead of the chainEntry map directly
+	for _, chainId := range gov.chainIds {
+		ce := gov.chains[chainId]
+		netUsage, _, incoming, err := sumValue(ce.transfers, startTime)
 		if err != nil {
-			// Don't return an error here, just return 0
-			return make([]*publicrpcv1.GovernorGetAvailableNotionalByChainResponse_Entry, 0)
+			// Report 0 available notional if we can't calculate the current usage
+			gov.logger.Error("GetAvailableNotionalByChain: failed to compute sum of transfers for chain entry",
+				zap.String("chainID", chainId.String()),
+				zap.Error(err))
+			resp = append(resp, &publicrpcv1.GovernorGetAvailableNotionalByChainResponse_Entry{
+				ChainId:                    uint32(ce.emitterChainId),
+				RemainingAvailableNotional: 0,
+				NotionalLimit:              ce.dailyLimit,
+				BigTransactionSize:         ce.bigTransactionSize,
+			})
+			continue
 		}
-		if value >= ce.dailyLimit {
-			value = 0
-		} else {
-			value = ce.dailyLimit - value
+
+		remaining := gov.availableNotionalValue(chainId, netUsage)
+
+		if !gov.flowCancelEnabled {
+			// When flow cancel is disabled, we expect that both the netUsage and remaining notional should be
+			// within the range of [0, dailyLimit]. Flow cancel allows flexibility here. netUsage may be
+			// negative if there is a lot of incoming flow; conversely, it may exceed dailyLimit if incoming
+			// flow added space, allowed additional transfers through, and then expired after 24h.
```

Comment:
I don't think this is true if flow cancel was on but is then switched off, right? I think keeping the warnings for this is fine (since this toggling of flow cancel from on to off is an edge case), but probably worth clarifying in the comment?